monkey/infection_monkey/network/info.py

```diff
@@ -21,6 +22,41 @@
 RTF_REJECT = 0x0200
 
 
+class IntRange:
+    """
+    Represents an inclusive range of integers, with a step size of 1.
+
+    Ensures that min <= max.
+    """
+
+    def __init__(self, min: int, max: int):
+        self._min = min
+        self._max = max
+        if min > max:
+            self._min, self._max = max, min
```

Comment:
I would go with `a, b` or similar instead of `min, max`, since `min` isn't guaranteed to actually be the minimum.
```suggestion
    def __init__(self, a: int, b: int):
        self._min = a
        self._max = b
        if a > b:
            self._min, self._max = b, a
```